botCommands/carlosQuote.js

```diff
@@ -1,17 +1,30 @@
+const Discord = require('discord.js');
 const { registerBotCommand } = require('../botEngine');
 
 const command = {
   regex: /(?<!\S)!cqg(?!\S)/,
-  cb: () => `**Here is a great quote about "Asking Questions in the TOP Discord"**
-    "Here's my take on asking questions here:
-    Naturally, we welcome them. But we will hold you accountable when you ask a question that is easily researched. Not because we think you're a jerk. But because independent research is so critical to your continued learning.
-    Imagine you are on your first job and you went to your lead dev for questions every few minutes. I'm sure they'd be happy to help but a better situation is if you do some initial research on your own then take what you've found to your lead if you still have questions.
-    That's a very different conversation from: "Hey, I have no idea about this. Please tell me everything."
-    vs
-    "So I did some research. This is what I understand. And this is what I don't understand. Can you help clarify this last bit?"
-    So yeah, feel free to continue to ask. But make sure you are doing your part by doing the research. This helps all of us grow."
-    **-Carlos**`,
+  cb: () => {
+    const template = new Discord.EmbedBuilder()
+      .setColor('#cc9543')
+      .setTitle('Asking Questions in the TOP Discord')
+      .setDescription(
+        'Here is a quote from Carlos about "Asking Questions in the TOP Discord"',
```

Comment:
Not certain if this is really necessary, especially with Carlos already in the footer, and the title making clear what the subject is? Looking great otherwise

Reply:
Yeah I agree, I tried all the options out of excitement and should have left that description out